syncd/scripts/syncd_init_common.sh

```diff
@@ -211,14 +211,28 @@ config_syncd_mlnx()
     DUAL_TOR="$(echo $SYNCD_VARS | jq -r '.dual_tor')"
     DSCP_REMAPPING="$(echo $SYNCD_VARS | jq -r '.dscp_remapping')"
 
-    # Make default sai.profile
+    SAI_COMMON_FILE_PATH=/usr/bin/sai-profile-common
+
+    if [[ -f $SAI_COMMON_FILE_PATH ]]; then
+        cat $SAI_COMMON_FILE_PATH > /tmp/sai.profile
+    fi
```

Comment:
@noaOrMlnx suggest moving this code block after specific SAI config part

Reply:
@nazariig this can not be done. we want the common file to first be applied, and then the specific SAI config to be applied, and if we have duplications, keep only the configurations from specific SAI file. 
Why do you think it's better to reverse the order?